packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -514,6 +514,28 @@ export const SectionStruct: Describe<SectionElement> = element('Section', {
   ),
 });
 
+export const NotificationChildrenStruct = children(
+  // eslint-disable-next-line @typescript-eslint/no-use-before-define
+  [lazy(() => NotificationComponentsStruct)],
+) as unknown as Struct<SnapsChildren<GenericSnapElement>, null>;
+
+/**
+ * A struct for the {@link BoxElement} type but with a limited subset of children.
+ */
+export const NotificationStruct: Describe<BoxElement> = element('Box', {
```

Comment:
Why do we need this?

It seems outdated too. It refers to `Box`, and accepts a bunch of props that we don't need for notifications.

Reply:
It's used to validate that the `Box` element's children are only the ones I've allowed.